x-pack/plugins/security_solution/scripts/run_cypress/utils.ts

```diff
@@ -90,25 +95,39 @@ export const parseTestFileConfig = (filePath: string): SecuritySolutionDescribeB
       'key.name',
       'ftrConfig',
     ]);
+    // @ts-expect-error
+    const devConfig = _.find(callExpressionProperties?.value?.properties, [
```

Comment:
nit: Maybe rename this local `devConfig` to avoid confusion and variable re-declaration TS error?

```suggestion
    const _devConfig = _.find(callExpressionProperties?.value?.properties, [
```